src/components/OptionButtons/OptionButtons.tsx

```diff
@@ -0,0 +1,12 @@
+import SortByButton from '../SortByButton';
+import styles from './OptionButtons.module.css';
+
+function OptionButtons() {
+   return (
+      <div className={styles.optionButtons}>
+         <SortByButton />
+      </div>
+   );
+}
+
+export default OptionButtons;
```

Comment:
Another idea is, what if someone would want to have this component, but without one of the buttons that this component will have? 
How can you solve this problem? (generalize the component, children props 😉 )